.github/workflows/ci-mobile-save-for-later.yaml

```diff
@@ -8,18 +8,21 @@ jobs:
       id-token: write
       contents: read
     steps:
-      - uses: actions/checkout@v3
-      - uses: aws-actions/configure-aws-credentials@v1
+      - uses: actions/checkout@v4
+      - uses: aws-actions/configure-aws-credentials@v2
         with:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
           aws-region: eu-west-1
-      - name: Set up JDK 8
-        uses: actions/setup-java@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v4
         with:
-          java-version: 8
-          distribution: temurin
+          java-version: 11
+          distribution: corretto
```

Comment:
Yeah, we should be using corretto everywhere ('cos it's AWS's distribution, optimised for AWS instances, and it's what AWS lambdas run on) 👍